packages/installer/src/recipe-builder.ts

```diff
@@ -3,7 +3,18 @@ import * as AddDependencyExecutor from "./executors/add-dependency-executor"
 import * as NewFileExecutor from "./executors/new-file-executor"
 import * as TransformFileExecutor from "./executors/file-transform-executor"
 
-export function RecipeBuilder() {
+export interface RecipeBuilder {
```

Comment:
externally, TS wasn't able to infer the `RecipeBuilder` type so I went ahead and added an explicit interface declaration. initially I tried converting it to a class instead of a plain JS object but the intellisense/tsserver experience was slightly worse and had less intelligent autocomplete so I went with this instead